en/frames_rover/differential_rover.md

```diff
@@ -0,0 +1,112 @@
+3. [RD_MAX_YAW_RATE](../advanced_config/parameter_reference.md#RD_MAX_YAW_RATE) [deg/s]: This is the maximum yaw rate you want to allow for your rover.
+  This will define the stick-to-yaw-rate mapping in acro mode as well as setting an upper limit for the yaw rate in mission mode.
+4. [RD_YAW_RATE_P](../advanced_config/parameter_reference.md#RD_YAW_RATE_P) and [RD_YAW_RATE_I](../advanced_config/parameter_reference.md#RD_YAW_RATE_I) [-]: Tuning parameters for the closed-loop yaw rate controller.
+
+   ::: info
+   This can be tuned by setting all previous parameters and then setting the rover to _acro mode_.
+   Use the right stick to yaw the rover on the spot and then observe the desired and actual yaw rate in the flight log.
+   Change parameters and iterate.
+
+   Suggestion: Start the tuning process with [RD_YAW_RATE_I](../advanced_config/parameter_reference.md#RD_YAW_RATE_I) equal to zero and only set if necessary.
+   :::
+
+This is enough to start using the rover in acro mode.
+To start driving mission the parameters in [Tuning (Mission)](#tuning-mission) also must be set.
+
+## Tuning (Mission)
+
+:::warning
+The parameters in [Tuning (Basic)](#tuning-basic) must also be set to drive missions!
+:::
+
+The module uses a control algorithm called pure pursuit, see [Mission Mode](../flight_modes_rover/index.md#mission-mode) for the basic tuning process.
+The additional parameters are separated into the following sections:
+
+### Mission Velocity
+
+These parameters tune velocity control in missions:
+
+- [RD_MISS_SPD_DEF](#RD_MISS_SPD_DEF): Sets the default velocity ($m/s$) for the rover during the mission.
+- [RD_MAX_ACCEL](#RD_MAX_ACCEL) ($m/s^2$) and [RD_MAX_JERK](#RD_MAX_JERK) ($m/s^3$) are used to calculate a velocity trajectory such that the rover comes to a smooth stop as it reaches a waypoint.
+- [RD_SPEED_P](#RD_SPEED_P) and [RD_SPEED_I](#RD_SPEED_I) are used to tune the closed-loop velocity controller during missions.
+
+### Yaw Rate
+
+The yaw rate setpoint is calculated by using the heading error calculated by the pure pursuit algorithm for a PID controller that can be tuned with [RD_HEADING_P](#RD_HEADING_P) and [RD_HEADING_I](#RD_HEADING_I).
+
+::: info
+There is some degree of overlap between this tuning and the pure pursuit controller gain set in [Mission Mode](../flight_modes_rover/index.md#mission-mode) as they both have an influence on how aggressive the rover will steer.
+:::
+
+### State Machine
+
+The module employs the following state machine to make full use of a differential-steering rovers ability to turn on the spot:
+![Differential state machine](../../assets/airframes/rover/rover_differential/differential_state_machine.png)
+
+These transition thresholds can be set with [RD_TRANS_DRV_TRN](#RD_TRANS_DRV_TRN) and [RD_TRANS_TRN_DRV](#RD_TRANS_TRN_DRV).
+
+### Parameters
+
+The following parameters affect the differential-steering rover in mission mode (overview):
```

Comment:
These are the mission params. Should we include ALL params here and reduce the heading level by one?